src/err.rs

```diff
@@ -63,15 +63,18 @@ pub struct NlmsghdrErr<T, P> {
 }
 
 /// Struct representing netlink packets containing errors
-#[derive(Debug, PartialEq, Eq, Size, FromBytesWithInput, ToBytes, Header)]
+#[derive(Debug, PartialEq, Eq, Size, FromBytesWithInput, ToBytes)]
 #[neli(from_bytes_bound = "T: NlType")]
-#[neli(from_bytes_bound = "P: FromBytesWithInput<Input = usize>")]
+#[neli(from_bytes_bound = "P: Size + FromBytesWithInput<Input = usize>")]
 pub struct Nlmsgerr<T, P> {
     /// Error code
     pub error: libc::c_int,
     /// Packet header for request that failed
-    #[neli(input = "input - Self::header_size()")]
+    #[neli(input = "input - std::mem::size_of::<libc::c_int>()")]
     pub nlmsg: NlmsghdrErr<T, P>,
+    #[neli(input = "input - std::mem::size_of::<libc::c_int>() - nlmsg.unpadded_size()")]
+    /// Contains attributes representing the extended ACK
+    pub ext_ack: GenlBuffer<u16, Buffer>,
```

Comment:
Should this be:
```rust
    pub ext_ack: GenlBuffer<NlmsgerrAttr, Buffer>,
```

with somewhere in consts:
```rust
// from enum nlmsgerr_attrs in <linux/netlink.h>
#[neli_enum(serialized_type = "u16")]
enum NlmsgerrAttr {
         Unused = 0,
         /// error message string (string)
         Msg = 1,
         /// offset of the invalid attribute in the original
         /// message, counting from the beginning of the
         /// header (u32)
         Offset = 2,
         /// arbitrary subsystem specific cookie to
         /// be used - in the success case - to identify a created
         /// object or operation or similar (binary)
         Cookie = 3,
         /// policy for a rejected attribute
         Policy = 4,
}
```

Reply:
Yes that is what I want the interface to look like. I mainly just wanted a prototype to see if it works for you first. It seems like it does, so I'll add this in!

Reply:
Resolved!